src/p5.dance.js

```diff
@@ -3,6 +3,9 @@
 const P5 = require('./loadP5');
 const Effects = require('./Effects');
 const replayLog = require('./replay');
+const SpritesheetBuilder = require('./SpritesheetBuilder');
+
+const VECTOR_MODE = window.location.search.startsWith('?vector=');
```

Comment:
Presumably this is meant to semi-imitate what we do with experiments in our main repo. Might be worth a short comment that this is a temporary hack. (NBD, but also worth noting that `?vector=false` will of course still result in VECTOR_MODE=true with this code).